src/com/intellij/idea/plugin/hybris/toolwindow/RemoteHacConnectionDialog.kt

```diff
@@ -144,6 +158,60 @@ class RemoteHacConnectionDialog(
                     .onChanged { urlPreviewLabel.text = generateUrl() }
                     .component
             }.layout(RowLayout.PARENT_GRID)
+            if (System.getProperty("os.name").lowercase(Locale.getDefault()).contains("win")) {
+                val distributions = WslDistributionManager.getInstance().installedDistributions
+                row {
+                    isWslCheckBox = checkBox("WSL")
+                        .selected(false)
```

Comment:
it looks like this is not persisted, yet, it should be part of the `RemoteConnectionSettings`.